crates/uv-distribution/src/metadata/requires_dist.rs

```diff
@@ -121,54 +117,55 @@ impl RequiresDist {
+                    )
+                    .map(|(name, requirements)| {
+                        let requirements = match source_strategy {
+                            SourceStrategy::Enabled => requirements
+                                .into_iter()
+                                .flat_map(|requirement| {
+                                    let group_name = name.clone();
+                                    let requirement_name = requirement.name.clone();
+                                    LoweredRequirement::from_requirement(
+                                        requirement,
+                                        &metadata.name,
+                                        project_workspace.project_root(),
+                                        project_sources,
+                                        project_indexes,
+                                        locations,
+                                        project_workspace.workspace(),
+                                        lower_bound,
+                                    )
+                                    .map(move |requirement| {
+                                        match requirement {
+                                            Ok(requirement) => Ok(requirement.into_inner()),
+                                            Err(err) => Err(MetadataError::GroupLoweringError(
+                                                group_name.clone(),
+                                                requirement_name.clone(),
+                                                Box::new(err),
+                                            )),
+                                        }
+                                    })
                                 })
-                            })
-                            .collect::<Result<Vec<_>, _>>(),
-                        SourceStrategy::Disabled => Ok(requirements
-                            .into_iter()
-                            .map(uv_pypi_types::Requirement::from)
-                            .collect()),
-                    }?;
-                    Ok::<(GroupName, Vec<uv_pypi_types::Requirement>), MetadataError>((
-                        name,
-                        requirements,
-                    ))
-                })
-                .collect::<Result<Vec<_>, _>>()?;
+                                .collect::<Result<Vec<_>, _>>(),
+                            SourceStrategy::Disabled => Ok(requirements
+                                .into_iter()
+                                .map(uv_pypi_types::Requirement::from)
+                                .collect()),
+                        }?;
+                        Ok::<(GroupName, Vec<uv_pypi_types::Requirement>), MetadataError>((
+                            name,
+                            requirements,
+                        ))
+                    })
+                    .collect::<Result<Vec<_>, _>>()?;
```

Comment:
No changes here; just indentation from moving to `FlatDependencyGroups::from_dependency_groups`.